doc/target.rst

```diff
@@ -450,3 +450,112 @@ Target
+
+    :param package_data_directory: This is the location of the data stored for
+        installed Android packages on the device.
+
+.. method:: Target.set_rotation(rotation)
+
+   Sets the screen orientation of the device as an integer value
+   representing the screen orientations. ``Natural : 0``, ``Rotated Left: 1``,
+   ``Inverted : 2`` and ``Rotated Right : 3``.
+
+.. method:: Target.get_rotation(rotation)
+
+   Returns an integer value representing the orientation of the devices screen.
+   ``0 : Natural``, ``1 : Rotated Left``,
+   ``2 : Inverted`` and ``3 : Rotated Right``.
+
+.. method:: Target.set_natural_rotation()
+
+   Sets the screen orientation of the device to its natural orientation.
+
+.. method:: Target.set_left_rotation()
+
+   Sets the screen orientation of the device to its 90 deg orientation.
+
+.. method:: Target.set_inverted_rotation()
+
+   Sets the screen orientation of the device to its inverted (180 deg)
+   orientation.
+
+.. method:: Target.set_right_rotation()
+
+   Sets the screen orientation of the device to its 270 deg orientation.
+
+.. method:: Target.set_auto_rotation(autorotate)
+
+   Specify whether the devices auto-rotation should be enabled.
+
+.. method:: Target.get_auto_rotation()
+
+   Returns ``True`` if the targets auto rotation is currently enabled and
+   ``False`` otherwise.
+
+.. method:: Target.set_airplane_mode(mode)
+
+   Specify whether the device should be in airplane mode.
+
+   .. note:: Requires the device to be rooted if the device is running Android 7+.
+
+.. method:: Target.get_airplane_mode()
+
+   Returns ``True`` if the target is currently in airplane mode and ``False``
+   otherwise.
+
+.. method:: Target.get_brightness()
+
+   Returns the devices current screen brightness as an integer between 0 and 255.
+
+.. method:: Target.set_brightness(value)
+
+   Sets the devices screen brightness as an integer between 0 and 255 to the
+   specified value.
+
+.. method:: Target.get_brightness()
+
+   Returns the devices current screen brightness as a percentage.
+
+.. method:: Target.get_auto_brightness()
+
+   Returns ``True`` if the targets auto brightness is currently enabled and
+   ``False`` otherwise.
+
+.. method:: Target.set_auto_brightness(auto_brightness)
+
+   Specify whether the devices auto brightness is currently enabled.
+
+.. method:: Target.ensure_screen_is_off()
+
+   Checks if the devices screen is on and if so turns it off.
+
+.. method:: Target.ensure_screen_is_on()
+
+   Checks if the devices screen is off and if so turns it on.
+
+.. method:: Target.is_screen_on()
+
+   Returns ``True`` if the targets screen is currently on and ``False``
+   otherwise.
+
+.. method:: Target.homescreen()
+
+   Returns the device to its home screen.
+
+.. method:: Target.swipe_to_unlock(direction=None)
+
+   Performs a swipe input on the device to try and unlock the device.
+   A direction of `horizontal` or `vertical` can be supplied to specify in
```

Comment:
I would also add `"diagonal"` as a valid value here and set it as the default (obviously, this relates to the actual function as well as the docs).